erpnext/assets/doctype/asset_repair/asset_repair.json

```diff
@@ -229,26 +230,30 @@
    "label": "Increase In Asset Life(Months)",
    "no_copy": 1
   },
-  {
-   "fieldname": "purchase_invoice",
-   "fieldtype": "Link",
-   "label": "Purchase Invoice",
-   "mandatory_depends_on": "eval: doc.repair_status == 'Completed' && doc.repair_cost > 0",
-   "no_copy": 1,
-   "options": "Purchase Invoice"
-  },
   {
    "fetch_from": "asset.company",
    "fieldname": "company",
    "fieldtype": "Link",
    "label": "Company",
    "options": "Company"
+  },
+  {
+   "fieldname": "section_break_rahs",
+   "fieldtype": "Section Break"
+  },
+  {
+   "fieldname": "asset_repair_purchase_invoices",
```

Comment:
always give shorter field names for tables like "invoices" to increase the usability and readability of the code.